src/ncp/spinel.c

```diff
@@ -2516,6 +2528,10 @@ const char *spinel_capability_to_cstr(unsigned int capability)
         ret = "NEST_TRANSMIT_HOOK";
         break;
 
+    case SPINEL_CAP_THREAD_SERVICE:
```

Comment:
Sorry nitpicking here but can you please move this after `SPINEL_CAP_THREAD_BORDER_ROUTER` to keep the order here match the order of definitions in header file (makes it easier to check later if all definitions are in the switch)

Reply:
Not a problem, shall do